packages/net/http/clientconnection.pony

```diff
@@ -1,6 +1,7 @@
 use "collections"
 use "net"
 use "net/ssl"
+use "net/tcp"
 
 actor _ClientConnection
```

Comment:
I think this file should be named `_client_connection.pony`, because the principal type declared in the file is `_ClientConnection`.